packages/react-core/src/components/DragDrop/DragDrop.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+
+interface ZoneNodes {
+  [zone: string]: HTMLElement[];
+}
+
+export const DragDropContext = React.createContext({
+  draggableZoneNodes: {} as ZoneNodes,
+  setDraggableZoneNodes: (_nodes: ZoneNodes) => {},
+  onDrop: (
+    _sourceDroppableKey: string | number,
+    _sourceDraggableKey: string | number,
+    _destDroppableKey: string | number | null,
+    _destDraggableKey: string | number | null
+  ) => false as boolean
+});
+
+interface DragDropProps {
+  /** Potentially Droppable and Draggable children */
+  children?: React.ReactNode;
+  /** Callback for drop event */
```

Comment:
Could you give a little more information about what all this function needs to do/what the various props are for? 
I think the example implementation could include comments for more insight, but this prop description could be a little more specific as well.

Reply:
It's a callback to actually move the elements in whatever the source of truth is.

I'm thinking of using indexes instead of ids since the first step to moving elements is almost always finding their index.